build.assets/Makefile

```diff
@@ -191,9 +191,9 @@ endif
 # generate gRPC code
 #
 .PHONY: grpc
-grpc: buildbox $(GRPC_PROTO_OUTPUTS)
+grpc: $(GRPC_PROTO_OUTPUTS)
 
-$(GRPC_PROTO_OUTPUTS): $(GRPC_PROTOS)
+$(GRPC_PROTO_OUTPUTS): buildbox $(GRPC_PROTOS)
```

Comment:
Moved the `buildbox` dependency to the list of dependencies for the proto output files - if left on the `grpc` target, the two dependencies might be built in parallel (`buildbox` and `$GRPC_PROTO_OUTPUTS`) while the latter depends on the first which leads to inadvertent modifications to the generated protobuf sources.